### ISSUE TITLE
Multiprocessed Mypal 28/29?

### ISSUE THREAD

**Issue by Thyristor-Chopper** [Feodor2/Mypal#116]
In about:config I set

![image](https://user-images.githubusercontent.com/76606635/236403915-0eb26f88-a992-42cb-8baa-72a63bdb8bc2.png)

and `dom.ipc.processCount;8`

restart Mypal, and boom.

![image](https://user-images.githubusercontent.com/76606635/236404004-61b7d2c2-62db-44f9-8a67-94145bfd7ff6.png)

well it's very broken, you'll get a title bar glitch so disable and re-enable the menu bar. and also about:xxx addresses won't work anymore. extensions won't work either. Right click menus won't show up. Don't try it in your main profile.

In Mypal 27.9.4 multiprocess won't work and only about:xxxx pages will break.

28.x is same as 29.x

**Comment by yotoprules** [comment]
Have you tried mypal 68?

**Comment by Thyristor-Chopper** [comment]
yes

**Comment by teknoskillz** [comment]
Yes, this is happening here with mine too, I have (4) mypal processes and one site I frequent daily broke as I was using it.
White pages appear when I use certain features, and the main navigation menu is not rendering so you cant log out. this is day 4 of it being logged in as soon as I visit the site. Older mypal versions like the stable release are not doing this.

I have multiple mypal versions installed, and the folders renamed....dunno of that could be the conflict?

**Comment by Thyristor-Chopper** [comment]
did you change the multiprocess config above?

you can reset it, i'll tell you how to do it if you say that this is the case

**Comment by Thyristor-Chopper** [comment]
and also this issue is actually just for an experiment, not a bug report... 🙂

**Comment by teknoskillz** [comment]
I dont see the exact same browser tab settings.

https://ibb.co/KwQVJfS

**Comment by Thyristor-Chopper** [comment]
you have to create it
